spec/system/decidim/admin/admin_creates_wallet_spec.rb

```diff
@@ -66,9 +66,43 @@
       it "doesn't show the information about receiving coins" do
         visit_steps_page
 
-        expect(page).not_to have_content("The usage of the Vocdoni platform has costs")
-        expect(page).not_to have_content("Test reseller")
-        expect(page).not_to have_css("input[value='#{wallet.private_key}']")
+        expect(page).to have_no_content("The usage of the Vocdoni platform has costs")
+        expect(page).to have_no_content("Test reseller")
+        expect(page).to have_no_css("input[value='#{wallet.private_key}']")
+      end
+    end
+
+    context "when prod environment" do
+      before do
+        allow(Decidim::Vocdoni).to receive(:api_endpoint_env).and_return("prod")
+        allow(Decidim::Vocdoni).to receive(:vocdoni_reseller_name).and_return("Test reseller")
+        allow(Decidim::Vocdoni).to receive(:vocdoni_reseller_email).and_return("test_reseller@example.org")
+      end
```

Comment:
This looks repeated from an example above, can we just add the "it" spec to the previous context "when prod environment"?